### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by pangia** [bitclout/identity-swift#7]
The description still says this is not ready. Is there a plan to finish this? Thanks.

**Comment by andyboyd** [comment]
Yup. Waiting on a few last changes to the identity web app before it can be finished, but should hopefully be very soon

**Comment by pangia** [comment]
Hi Andy. Any update?  I can help test this out if needed. Thanks!

**Comment by ismyhc** [comment]
Curious about this too :) Ive actually just recently started a swift package to interact with the DeSo backend api. Still a work in progress :).

https://github.com/Galaxoid-Labs/DeSoKit

**Comment by pangia** [comment]
Hi Andy. I am going to use this package as part of an Objective C project (long story.. I don't have any other option). I have pulled the code and made some initial changes to get it to build in my project, but am still missing a few parts.. mostly due to my not fully understanding the flow. Could you possibly give a high level overview of how the package is intended to be called? Then I can open up the correct functions to be visible in objc. Once I get it all working I will document what I have done for future devs stuck in this same predicament. Thanks, Nick.

**Comment by nicholasraphael** [comment]
@andyboyd Hi, new to native ios and going off of what was said above could we get minimal usage docs for those that want to experiment with this just to get us started

**Comment by pangia** [comment]
I see @ismyhc has started making some updates to this lib. Great news!

Unfortunately because my main project is Objective C I am unable to use this lib (SPM doesn't work well with Obj C projects.. and when I tried to go outside of SPM it got too bloated).

Anyways... I started building an Obj C identity framework. I will create a new repo once I get it cleaned up and working.

**Comment by ismyhc** [comment]
> I see @ismyhc has started making some updates to this lib. Great news!
> 
> Unfortunately because my main project is Objective C I am unable to use this lib (SPM doesn't work well with Obj C projects.. and when I tried to go outside of SPM it got too bloated).
> 
> Anyways... I started building an Obj C identity framework. I will create a new repo once I get it cleaned up and working.

Hey @pangia that's unfortunate. Im most comfortable in Swift which is why I reached out here to help. Im not super familiar with Objective-c, but I can help you if you run into any issues.
